torch/csrc/jit/codegen/cuda/lower2device.cpp

```diff
@@ -226,15 +225,40 @@ void GpuLower::lower(Fusion* fusion, DataType index_type) {
 
   FusionGuard fg(fusion_);
   // prepare for lowering
+  if (isDebugDumpEnabled(DebugDumpOption::PassTransform)) {
```

Comment:
Please pull this feature out as a separate PR

Reply:
Done.
debug_lower_GPU PR: https://github.com/azazhu/pytorch/tree/fw/dump_debug
frontend PR: https://github.com/azazhu/pytorch/tree/fw/idx_sel_frontend
backend PR: https://github.com/azazhu/pytorch/tree/fw/idx_sel_backend